docs/_future/triggers.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Triggers
+order: 7
+slug: triggers
+lang: en
+layout: tutorial
+permalink: /future/triggers
+---
+# Triggers <span class="label-beta">BETA</span>
+
+<div class="section-content">
+Triggers are but one of the three building blocks that make up next-generation Slack apps. You will encounter all three as you walk the path of building your Slack app:
+
+1. Functions are what define the actions of your app
+2. Workflows are made up of Functions
+3. Triggers are how Workflows are executed (⬅️ you are here)
+
+Since Triggers are what trigger your Workflows, you need to have a Workflow before you can create a Trigger. Acquaint yourself with the [documentation on Workflows](/bolt-js/future/workflows), then head back here. We'll wait!
+
+With the knowledge of Workflows within your noggin, let's take a look at the different types of Triggers, and how you can implement them into your new app. 
+
+You will come to many forks in this metaphorical road that is Trigger implementation. There are no wrong choices; all roads lead to your own next-generation Slack app. 
```

Comment:
I figure this is straight from the api site and I'd leave that stuff there - IMO we can cut this line since it feels not so much to the point.